nebula-components/tools/_triangle.scss

```diff
@@ -1,32 +1,27 @@
-@mixin nb-triangle($direction, $size: $nb-flyout-triangle-size, $color: $nb-flyout-triangle-color) {
+@mixin nb-triangle(
+  $direction,
+  $color: $nb-color-white,
+  $transform: rotate(-45deg),
+  $size: ($nb-spacing-unit / 2),
+  $box-shadow-color: $nb-color-grey-light) {
   content: '';
   display: block;
   position: absolute;
   height: 0;
   width: 0;
+  transform-origin: 0 0;
+  transform: #{$transform};
+  border-width: $size;
+  border-style: solid;
   @if ($direction == up) {
-    bottom: 100%;
-    border-bottom: $size solid $color;
-    border-left: $size solid transparent;
-    border-right: $size solid transparent;
+    bottom: 96.75%;
+    box-shadow: 1px -1px 0 $box-shadow-color;
+    border-color: $color $color transparent transparent;
   }
   @else if ($direction == down) {
-    top: 100%;
-    border-top: $size solid $color;
-    border-left: $size solid transparent;
-    border-right: $size solid transparent;
-  }
-  @else if ($direction == left) {
-    right: 100%;
-    border-top: $size solid transparent;
-    border-bottom: $size solid transparent;
-    border-right: $size solid $color;
-  }
-  @else if ($direction == right) {
-    left: 100%;
-    border-top: $size solid transparent;
-    border-bottom: $size solid transparent;
-    border-left: $size solid $color;
+    top: 99.75%;
+    box-shadow: -1px 1px 0 $box-shadow-color;
```

Comment:
Good use of box shadow here, however I don't think the variable name needs to know what technique it is using to add this _second_ border.  so I would use `nb-color-base-border` bearing in mind that we went for borders on the `c-card` component and others on Friday.